src/assets/sprites/icon-checked-16.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
In order to be more coherent, we don't need to specify the icon size in the file name here, just `icon-checked.svg` just be enough.